README.md

```diff
@@ -1,31 +1,39 @@
-# FfthhStoreAng2
+# FFTHH Store UI
+A basic Angular UI originally generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.19.
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.19.
+## Getting Started
+This project makes use of Docker and VS Code to provide a portable and explicit environment to develop against and target deployments towards.
 
-## Development server
+### Workstation Dependencies
+Ensure you have the following installed on your workstation:
 
-Run `yarn start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+*   [Docker Desktop](https://www.docker.com/products/docker-desktop)
+*   [Visual Studio Code](https://code.visualstudio.com/download)
+*   [Remote Containers Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
 
-## Code scaffolding
+### Running Locally
+1. Clone down this repository.
+2. Open the folder in VS Code, the `Remote Containers` extension will notice the `.devcontainer\devcontainer.json` and attempt to build a container to house this environment and open this folder inside the container, as described by the project's root `Dockerfile`.
+3. Once VS Code is running with the folder hosted out of the container, set the application config in the `./environments/environments.ts` file.
+4. Run this application by running `yarn start` from the container terminal and navigate to `http://localhost:53529/`. Debug/inspect the application using Chrome Developer Tools (`Ctrl+Shift+I`). The app will automatically reload if you change any of the source files. 
```

Comment:
Step 1 and 2 worked flawlessly.  Step 3 is a bit vague, is their specific application config that needs to be set, or is this just a general comment about where to put environment config?   Step 4 is working quite right for me, `yarn start` does build and serve the project but I'm unable to view the angular app at the specified path or at `:4200`.

Reply:
Was able to get this working on port `4200` by modifying the `start` script to `ng serve --host 0.0.0.0`.

Reply:
I was also able to fix this by serving to a different port than `4200` and then setting up a temporary forwarded port via the Remote Container window in VS Code.

Reply:
Love, it thanks for the feedback!